rspace/src/main/scala/coop/rchain/rspace/LMDBStoreScodec.scala

```diff
@@ -0,0 +1,375 @@
+package coop.rchain.rspace
+
+import java.nio.ByteBuffer
+import java.nio.file.Path
+import java.security.MessageDigest
+
+import cats.implicits._
+import coop.rchain.rspace.internal._
+import coop.rchain.rspace.util._
+import org.lmdbjava.DbiFlags.MDB_CREATE
+import org.lmdbjava._
+
+import coop.rchain.rspace.scodecmodels._
+import scala.collection.JavaConverters._
+import scodec.Codec
+import scodec.bits._
+
+import coop.rchain.rspace.scodecmodels.rscodecs._
+
+/**
+  * The main store class.
+  *
+  * To create an instance, use [[LMDBStoreScodec.create]].
+  */
+class LMDBStoreScodec[C, P, A, K] private (env: Env[ByteBuffer],
```

Comment:
As we discussed offline, let's go ahead and rename this to `LMDBStore`, replacing the existing implementation. 

Also, please make sure you delete any existing protobuf-related code left behind.